### PR TITLE
Support getters and setters with @Input and @Output refs #83

### PR DESCRIPTION
**Pull request by MichaelRFairhurst**
Just a matter of supporting MethodDeclaration members as well, provided
each .isGetter or .isSetter. Report errors otherwise.

### CODE REVIEW

File: analyzer_plugin/test/tasks_test.dart [hunk 1]
```diff
@@ -523,6 +534,8 @@ class MyComponent {
   EventEmitter<int> outputThree;
   @Output('outputFour')
   EventEmitter fourthOutput;
+  @Output()
+  EventEmitter get someGetter() => null;
```
Comment (scheglov) [inline analyzer_plugin/test/tasks_test.dart]: This does not look as a valid Dart code to me.

File: analyzer_plugin/test/tasks_test.dart [hunk 2]
```diff
@@ -616,6 +640,46 @@ class B {}
...
+  }
+
+  void test_outputOnSetterIsError() {
+    String code = r'''
+import '/angular2/angular2.dart';
+
+@Component(selector: 'my-component')
+class MyComponent {
+  @Output()
+  set someSetter(x) => null;
```
Comment (scheglov) [inline analyzer_plugin/test/tasks_test.dart]: I'd use `set someSetter(x) {}` for setter.

File: analyzer_plugin/test/tasks_test.dart [hunk 3]
```diff
@@ -414,6 +414,8 @@ class MyComponent {
   String firstField;
   @Input('secondInput')
   String secondField;
+  @Input()
+  String set someSetter() => null;
```
Comment (scheglov) [inline analyzer_plugin/test/tasks_test.dart]: I think every setter in Dart should have exactly one parameter and cannot have a return type (or void).

**Comment by MichaelRFairhurst** [comment]
Great! Missed one of the two "someSetter"s, will merge once travis confirms I didn't break anything in that oneline change.

**Pull request merged**